main/conceptual/js-programming/eventual-send.md

```diff
@@ -1,4 +1,4 @@
-# eventual send with `E()`
+# Eventual Send with E()
```

Comment:
hm. why remove the `E()` formatting?

Reply:
I really don't like how the docs use the code snippet formatting (`E()`) for atomic programming items (e.g., function names, object names, etc.) I prefer to reserve that formatting for entire programming ideas (e.g., a proper function call: `E(zoe).getBrands(myInstance)`), and I used bold for atomic programming items. I find it much more readable & usable that way. Also, I prefer to make the first occurrence of an atomic item a link to the reference docs where the item is described.
 But then Rowland's request for the PSM docs reshuffled my priorities, and I didn't go through the docs & universally fix the formatting. I'll do so sometime in the future. (I should prolly change that formatting back to `E()` for consistency's sake, but I'm being lazy.)

Reply:
ok.